chalicelib_smaht/checks/helpers/utils.py

```diff
@@ -96,7 +96,7 @@ def make_embed_request(ids, fields, connection):
         fields = [fields]
     id_chunks = chunk_ids(ids, chunk_size=5)  # Max 5 IDs to /embed as of 20220601 -drr
     for id_chunk in id_chunks:
```

Comment:
Some reason id_chunks has a variable name? I sometimes do this for debugging but put it back for the iteration.